rubicon-htb.js

```diff
@@ -10,6 +10,10 @@
  * prior written permission of Index Exchange.
  */
 
+/* eslint no-eval: 0 */
+/* eslint no-console: ["error", { allow: ["warn", "error"] }] */
```

Comment:
As we recommend against using console calls, please do not overwrite this restriction.